src/analysis/retail/evoker/shared/constants.ts

```diff
@@ -14,6 +14,8 @@ export const BASE_EVOKER_RANGE = 25;
 export const CLOBBERING_SWEEP_CDR = 45;
 export const HEAVY_WINGBEATS_CDR = 45;
 
+export const TIME_SPIRAL_BASE_DURATION = 10;
```

Comment:
It would make more sense for this const to be in milliseconds.
```suggestion
export const TIME_SPIRAL_BASE_DURATION_MS = 10_000;
```